modules/misc/qt.nix

```diff
@@ -194,20 +191,9 @@ in {
 
     home = {
       sessionVariables = envVars;
-      # home.sessionVariables does not support setting the same environment
-      # variable to different values.
-      # Since some other modules may set the QT_PLUGIN_PATH or QML2_IMPORT_PATH
-      # to their own value, e.g.: fcitx5, we avoid conflicts by setting
-      # the values in home.sessionVariablesExtra instead.
-      sessionVariablesExtra = ''
-        export QT_PLUGIN_PATH=${envVarsExtra.QT_PLUGIN_PATH}
-        export QML2_IMPORT_PATH=${envVarsExtra.QML2_IMPORT_PATH}
-      '';
+      sessionSearchVariables = envVarsExtra;
     };
 
-    # Apply theming also to apps started by systemd.
-    systemd.user.sessionVariables = envVars // envVarsExtra;
```

Comment:
Removing this since someone pointed that this is already done by `xsession.importedVariables`: https://github.com/nix-community/home-manager/pull/4677#discussion_r1399643300